platine-management-service/src/main/java/fr/insee/survey/datacollectionmanagement/metadata/service/impl/CampaignServiceImpl.java

```diff
@@ -237,10 +237,9 @@ private CollectionStatus getCollectionStatus(String campaignId) {
     }
 
     @Override
-    public CampaignCommonsOngoingDto findCampaignOngoingDtoById(String campaignId) {
+    public CampaignCommonsDto findCampaignDtoById(String campaignId) {
         return campaignRepository.findById(campaignId)
```

Comment:
Du coup utiliser la méthode this.findById pour récupérer la campagne et ensuite mapper l'objet. Surtout que finalement la différence entre les endpoints GET /api/campaigns/commons/{id} et GET /api/campaigns/{id} , c'est juste la représentation de la donnée